ml/src/test/scala/ai/h2o/sparkling/ml/models/H2OMOJOModelTestSuite.scala

```diff
@@ -218,6 +219,8 @@ class H2OMOJOModelTestSuite extends FunSuite with SharedH2OTestContext with Matc
     val gbm = configureGBMForProstateDF()
 
     val model = gbm.fit(trainingDF)
+    val s = model.getModelDetails()
+    val json = new Gson().fromJson(s, classOf[JsonObject])
```

Comment:
There is no any assertion using the json entity. What's the purpose?

Reply:
This line is left-over from debugging during the PR development, will remove that.